examples/solid-client-access-grants-demo/package.json

```diff
@@ -17,8 +17,8 @@
     "start": "node ."
   },
   "dependencies": {
-    "@inrupt/solid-client-access-grants": "file:../../",
-    "@inrupt/solid-client-authn-node": "^1.12.1",
+    "@inrupt/solid-client-access-grants": "^2.3.2",
```

Comment:
I dont think we should be targetting a released version of this lib since part of the usefulness of this demo app is for developers to use it while developing new features/fixes and running it with the demo to test out new things.